### ISSUE TITLE
fatal error: phpunit unable to locate functions.php

### ISSUE THREAD

**Issue by EvanHerman** [wp-cli/wp-cli#2579]
Trying to run phpunit on a plugin, but the functions.php file required by phpunit is not found. Re-provisioning does not seem to resolve the problem as suggested in #2259 

 I have tried re-provisioning, vagrant destroy/vagrant up --provision - but the phpunit functions.php file is not being re-created. 

> PHP Fatal error:  require_once(): Failed opening required '/srv/www/wordpress-develop/tests/phpunit//includes/functions.php' (include_path='.:/usr/share/php:/usr/share/pear') in /srv/www/-unit-tests/htdocs/wp-content/plugins/mailchimp-extender/tests/bootstrap.php on line 14

Not sure what really happened here. I had this running yesterday and was running unit tests without issue.

Any suggestions?

**Comment by danielbachhuber** [comment]
Does `/srv/www/wordpress-develop` exist? What's in it? How was it created originally?

> I have tried re-provisioning, vagrant destroy/vagrant up --provision - but the phpunit functions.php file is not being re-created.

This sounds like a problem with your provisioning system? Are you using a publicly-accessible Vagrant?

> I had this running yesterday and was running unit tests without issue.

What did you do between then and now?

**Comment by EvanHerman** [comment]
Hey Daniel,

Thanks for the response. I opened up a ticket over on vvv, and was able to resolve things after some additional tinkering.

https://github.com/Varying-Vagrant-Vagrants/VVV/issues/856

In short, I deleted the WordPress-* directories, and re-provisioned and the directories were in fact created properly. For whatever reason, WordPress wasn't installed to the WordPress-Develop directory after provisioning, so I copied over the files from WordPress-Default and things began to work again as expected.

Not sure why WordPress wasn't installed, but the deleting and re-provisioning did in fact seem to resolve the directory creation issue.
